lib/tests/modules/disable-define-enable.nix

```diff
@@ -0,0 +1,5 @@
+{ lib, ... }:
+
+{
+  disabledModules = [ ./define-enable.nix ];
```

Comment:
nit: Change `lib/tests/modules/default.nix` to add a `modulesPath` special argument, and use it to double check that we can lookup for modules as the string `"define-enable.nix"` instead of a path.